src/main/java/com/robin/hoteless/App/Greeting.java

```diff
@@ -0,0 +1,20 @@
+package com.robin.hoteless.App;
```

Comment:
Another note:

Package names should be camelCase
Class names should be CapitalCamelCase
Method names should be camelCase
Field (Class variable) and local var names should be camelCase
Constants (final String) should be ALL_CAPS_WITH_UNDERSCORES